docs/tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+Tutorials
+===========================
+Feel free to delve into these tutorials and unlock the features in just a matter of minutes. 
+
+
+| Feature |Task |Model| Notebook Link        |
+| ------- | --------------------------------------------- | ----------------- | -------------------- |
+|Static and dynamic post training quantization|language-modeling|Bert-Base|[Notebook](./pytorch/language-modeling/bert-base-uncased.ipynb)|
+|Static and dynamic post training quantization|Multiple Choice|Bert-Base|[Notebook](./pytorch/multiple-choice/bert-base-uncased_SWAG.ipynb)|
+|Static and dynamic post training quantization|Question Answering|MiniLM (Distilled from RoBERTa Large)|[Notebook](./pytorch/question-answering/Dynamic_MiniLM_SQuAD.ipynb)|
+|Static and dynamic post training quantization|Question Answering|Bert Large|[Notebook](./pytorch/question-answering/bert-large-uncased-whole-word-masking-finetuned-squad.ipynb)|
+|Dynamic post training quantization|Summarization|Bert Base|[Notebook](./pytorch/summarization/pegasus-samsum.ipynb)|
+|Static and dynamic post training quantization|Token Classification|Distilbert Base|[Notebook](./pytorch/token-classification/distilbert_base_ner.ipynb)|
+|Dynamic post training quantization|Translation|T5 small|[Notebook](./pytorch/translation/t5-small.ipynb)|
+|Static and dynamic post training quantization|Text Classification|Bert Base|[Notebook](./pytorch/text-classification/bert-base-uncased-MRPC.ipynb)|
+|Pruning|Text Classification|Bert base|[Notebook](./pytorch/text-classification/pruning.ipynb)|
+|Distillation|Question Answering|enter the model name|[Notebook](./pytorch/question-answering/distillation.ipynb)|
```

Comment:
enter the model name - what does it mean?

Reply:
user need to input model name or path

Reply:
Specified by user